src/core/qgsogrutils.cpp

```diff
@@ -20,6 +20,7 @@
 #include "qgsfields.h"
 #include <QTextCodec>
 #include <QUuid>
+#include <QtDebug>
```

Comment:
Please don't use qDebug outside of unit tests. Use the QgsDebugMsg methods instead, as the whole purpose of building with debug is to avoid debug outputs like this!